docs/docs-content/palette-cli/commands/validator.md

```diff
@@ -50,11 +54,13 @@ can install the Validator into an existing Kubernetes cluster by using the Helm
 
 The `install` subcommand accepts the following flags.
 
-| **Short Flag** | **Long Flag**   | **Description**                                                                                               | **Type** |
-| -------------- | --------------- | ------------------------------------------------------------------------------------------------------------- | -------- |
-| `-f`           | `--config-file` | Install the Validator using a configuration file (optional). Provide the file path to the configuration file. | string   |
-| `-o`           | `--config-only` | Generate a configuration file without proceeding with an actual install. Default: false                       | boolean  |
-| `-h`           | `--help`        | Help with any command.                                                                                        | -        |
+| **Short Flag** | **Long Flag**   | **Description**                                                                                                                                 | **Type** |
+| -------------- | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `-f`           | `--config-file` | Install the Validator using a configuration file (optional). Provide the file path to the configuration file.                                   | string   |
+| `-o`           | `--config-only` | Generate a configuration file without proceeding with an actual install. Default: false                                                         | boolean  |
+| `-h`           | `--help`        | Help with any command.                                                                                                                          | -        |
+| `-r`           | `--reconfigure` | Reconfigure Validator and plugins prior to the installation. The `--config-file` flag must be included. Default: false.                         | boolean  |
+| `-p`           | `--password`    | Update credentials provided in a configuration. This does proceed with installation. The `--config-file` flag must be included. Default: false. | boolean  |
```

Comment:
```suggestion
| `-p`           | `--password`    | Update credentials provided in the configuration file. This does proceed with installation. The `--config-file` flag must be included. Default: false. | boolean  |
```

Reply:
> This does proceed with installation.

I'm not clear on what this applies to 🤔 Meaning, if you use this flag, the CLI proceeds with installation?